flexget/plugins/cli/seen.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import unicode_literals, division, absolute_import
+
+from flexget import options
+from flexget.event import event
+from flexget.logger import console
+from flexget.manager import Session
+from flexget.plugins.filter import seen
+from flexget.utils.imdb import is_imdb_url, extract_id
+
+
+def do_cli(manager, options):
+    if options.seen_action == 'forget':
+        seen_forget(manager, options)
+    elif options.seen_action == 'add':
+        seen_add(options)
+    elif options.seen_action == 'search':
+        seen_search(options)
+
+
+def seen_forget(manager, options):
+    forget_name = options.forget_value
+    if is_imdb_url(forget_name):
+        imdb_id = extract_id(forget_name)
+        if imdb_id:
+            forget_name = imdb_id
+
+    count, fcount = seen.forget(forget_name)
+    console('Removed %s titles (%s fields)' % (count, fcount))
+    manager.config_changed()
+
+
+def seen_add(options):
+    seen_name = options.add_value
+    if is_imdb_url(seen_name):
+        imdb_id = extract_id(seen_name)
+        if imdb_id:
+            seen_name = imdb_id
+    seen.add(seen_name, 'cli_add', {'cli_add': seen_name})
+    console('Added %s as seen. This will affect all tasks.' % seen_name)
+
+
+def seen_search(options):
+    with Session() as session:
```

Comment:
hmm, why add session here? why not add it to seen.search function with @with_session